resources/managed/all-osd-resources.yaml

```diff
@@ -0,0 +1,419 @@
+ConfigMap:
```

Comment:
I believe we need to have a single root node that these fall under, otherwise these are all considered separate yaml documents.